reference/datasets.mdx

```diff
@@ -144,7 +144,9 @@ To delete a dataset, follow these steps:
 
 ## Change data retention period
 
-The data retention period determines how long Axiom stores your data. By default, the data retention period is defined by your pricing plan and is the same for all datasets. You can configure custom retention periods for individual datasets. Custom retention periods can only be shorter than your pricing plan’s default retention period. If you need a longer retention period, [contact Support](https://axiom.co/contact).
+The data retention period determines how long Axiom stores your data. By default, the data retention period is defined by your pricing plan and is the same for all datasets. You can configure custom retention periods for individual datasets. As a result, Axiom automatically deletes data after the specified time period instead of the default one defined by your pricing plan.
```

Comment:
```suggestion
The data retention period determines how long Axiom stores your data. By default, the data retention period is defined by your pricing plan and is the same for all datasets. You can configure custom retention periods for individual datasets. As a result, Axiom automatically trims data after the specified time period instead of the default one defined by your pricing plan.
```